cli.js

```diff
@@ -201,6 +201,7 @@ function init(files) {
 function handlePaths(files) {
 	if (files.length === 0) {
 		files = [
+			'!node_modules',
```

Comment:
I think we can use `'!**/node_modules',` to match `node_modules` at any level and not having to do the filtering below.